tests/conftest.py

```diff
@@ -30,3 +32,11 @@ def mock_db_session():
     db_session.add = MagicMock()
     db_session.refresh = MagicMock()
     yield db_session
+
+
+@pytest.fixture()
+def mock_item_list():
+    items = []
+    items.append(schemas.ItemBase(uri="/some/path", object_key="abcde"))
+    items.append(schemas.ItemBase(uri="/other/path", object_key="a1b2"))
```

Comment:
```suggestion
    items = [
        schemas.ItemBase(uri="/some/path", object_key="abcde"))
        schemas.ItemBase(uri="/other/path", object_key="a1b2"))
    ]
```
^ should be a little more efficient, no?

Reply:
Agree. It should be.